core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -659,7 +659,10 @@ class ContainerPool(
 
     private def getDockerImageName(action: WhiskAction)(implicit transid: TransactionId): String = {
         // only Exec instances that are subtypes of CodeExec reach the invoker
-        val imageName = ExecImageName.containerImageName(config.dockerRegistry, config.dockerImagePrefix, action.exec.asInstanceOf[CodeExec[_]], config.dockerImageTag)
+        val exec = action.exec.asInstanceOf[CodeExec[_]]
```

Comment:
We need to introduce a new type in the invoker so that it is dealing with code exec can action vs the more generic exec. Maybe something for the new invoker?